app/Models/User.php

```diff
@@ -3,6 +3,9 @@
 namespace App\Models;
 
 // use Illuminate\Contracts\Auth\MustVerifyEmail;
```

Comment:
Leaving commented lines everywhere is like covering your roomate's cat with post it notes about your internal thoughts on a dungeons and dragons game that nobody else was a part of. Nobody knows why they're there, what they means, or why you fell asleep on the couch covered in cheetos dust and won't wake up. It makes your random thoughts and inconsiderate brain artifacts everyone else's problem.

Reply:
Comment & unused `use` statements removed!